src/messages/layouts/MessageLayoutContainer.cpp

```diff
@@ -369,18 +369,18 @@ void MessageLayoutContainer::paintSelection(QPainter &painter, int messageIndex,
                     {
                         returnAfter = true;
                         index = line.startCharIndex;
-                        for (int i = line.startIndex; i < line.endIndex; i++)

```

Comment:
Renaming the variables doesn't improve readability. An actual change in the code structure with less nesting would be needed for that.